docs_src/security/Ch-Secure-Consul.md

```diff
@@ -0,0 +1,102 @@
+# Secure Consul
+
+!!! edgey "EdgeX 2.0"
+
+## Introduction
+
+In the current EdgeX architecture, `Consul` is pre-wired as the default agent service mainly for
```

Comment:
```suggestion
In the current EdgeX architecture, `Consul` is pre-wired as the default agent service for
```

Reply:
i am ok to remove `mainly`. the main reason I added mainly is that Consul can also do `Service Discovery` as it serves `service mesh` agent in that case but we don't use that feature in EdgeX; so just FYI